EpiJS/plots.js

```diff
@@ -71,7 +71,7 @@ function plot(model, time, name, title='Cases vs. Time') {
             mode: 'lines'
         })
     }
-
+    /* istanbul ignore if */
```

Comment:
Again here ... and later in the same file

Reply:
We can't test things running on the web.